resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/R/Dockerfile

```diff
@@ -19,10 +19,10 @@ ARG base_img
 FROM $base_img
 WORKDIR /
 RUN mkdir ${SPARK_HOME}/R
-COPY R ${SPARK_HOME}/R
 
 RUN apk add --no-cache R R-dev
 
+COPY R ${SPARK_HOME}/R
```

Comment:
Is this change intentional?

Reply:
This makes the docker build not run the `R` and `R-dev` installations each time an update to the jar is made. This is a minor change that helps with dev :)